pkg/agent/interfacestore/types.go

```diff
@@ -72,6 +72,7 @@ type InterfaceStore interface {
 	GetContainerInterface(podName string, podNamespace string) (*InterfaceConfig, bool)
 	GetNodeTunnelInterface(nodeName string) (*InterfaceConfig, bool)
 	GetContainerInterfaceNum() int
+	GetContainerInterfaces() []*InterfaceConfig
```

Comment:
Do you mean make the command like `antctl get pod podName -n nsName` ?
Then if user doesn't specify the namespace name, let's show all pods with different namespaces ?

Reply:
@lzhecheng For CRD, currently we only have the field LocalPodNum to show the number of Pod. I did not distinguish them by namespaces. I can enhance this part if we want more information for local pods in CRD. But if the number of Pods is too large, I am afraid the CRD is not friendly way to show.

However, CLI can show more information than CRD if we want. Since this patch has already added a separate function GetContainerInterfaces to get details, we can get Pod namespace from InterfaceConfig if we would like to show it via CLI.